syft/pkg/cataloger/common/cpe/generate.go

```diff
@@ -83,7 +83,7 @@ func candidateVendors(p pkg.Package) []string {
 	// allow * as a candidate. Note: do NOT allow Java packages to have * vendors.
 	switch p.Language {
 	case pkg.Ruby, pkg.JavaScript:
-		vendors.addValue("*")
+		vendors.addValue(wfn.Any)
```

Comment:
Nice find

Reply:
Thanks... that one took a little while to find and is one of the main reasons why I beefed up the encode-decode cycle tests for all of this.